src/LeanCode.Contracts/Security/AuthorizeWhenAttribute.cs

```diff
@@ -4,23 +4,15 @@
 namespace LeanCode.Contracts.Security;
 
 [AttributeUsage(AttributeTargets.Class | AttributeTargets.Interface, AllowMultiple = true, Inherited = true)]
-public abstract class AuthorizeWhenAttribute : Attribute
+public abstract class AuthorizeWhenAttribute(Type authorizerType, object? customData = null) : Attribute
 {
-    private readonly Type authorizerType;
-    private readonly object? customData;
-
-    protected AuthorizeWhenAttribute(Type authorizerType, object? customData = null)
-    {
-        this.authorizerType = authorizerType;
-        this.customData = customData;
-    }
+    private readonly Type authorizerType = authorizerType;
+    private readonly object? customData = customData;
 
     public static List<AuthorizerDefinition> GetCustomAuthorizers<T>() => GetCustomAuthorizers(typeof(T));
 
-    public static List<AuthorizerDefinition> GetCustomAuthorizers(Type type)
-    {
-        return type.GetCustomAttributes<AuthorizeWhenAttribute>().Select(AuthorizerDefinition.Create).ToList();
-    }
+    public static List<AuthorizerDefinition> GetCustomAuthorizers(Type type) =>
+        [.. type.GetCustomAttributes<AuthorizeWhenAttribute>().Select(AuthorizerDefinition.Create)];
```

Comment:
Not a fan of this syntax. Also this space after `..` is weird for me, but that's probably a csharpier thing.

Reply:
The space is indeed a CSharpier thing. Weird, but I don't hate it.

(Take a look at how CSharpier formats XML files like csprojs to see something truly cursed :see_no_evil:)

As for `[.. foo]` vs `foo.ToBar()`, I'm not opposed to keeping the latter but the former does have one major advantage: it does the right thing with a single syntax, for many collection types, as long as the type can be inferred from context. But maybe there's enough space to keep both of them, without favoring one over another?

Reply:
> What do you mean? I've used csharpier 1.0.x somewhere, and csprojs seemed fine? 🤨

https://github.com/leancodepl/contractsgenerator/blob/cf1ac680e71ce1e5ff7a88861f9e85cb1b798561/examples/project/Directory.Build.targets#L3-L5